docker/Dockerfile-pgbase

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
+RUN apt-get update && apt-get install -y \
+    libffi-dev \
+    libssl-dev \
+    postgresql-client
+ENV PYTHONBUFFERED 1
+ENV DATABASE_URL postgres://postgres:postgres@db:5432/postgres
+ENV FLASK_SECRET_KEY GBhQrZfbN9KGMcyLVImGeS2Wtj3SFCdF
+ENV RABBIT_MANAGEMENT_URL http://pulse:15672/api/
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
```

Comment:
Nit: For these simpler cases, whilst `ADD` has the same effect as `COPY`, the latter is preferred for clarity:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy